requirements.txt

```diff
@@ -15,27 +15,28 @@ google-api-python-client==2.116.0
 ibm_cloud_sdk_core==3.18.0
 ibm_vpc==0.20.0
 jinja2==3.1.4
-krkn-lib==2.1.3
+krkn-lib==2.1.4
 lxml==5.1.0
-kubernetes==26.1.0
+kubernetes==28.1.0
 oauth2client==4.1.3
 pandas==2.2.0
 openshift-client==1.0.21
 paramiko==3.4.0
 podman-compose==1.0.6
+podman-compose==1.0.6
 pyVmomi==8.0.2.0.1
 pyfiglet==1.0.2
 pytest==8.0.0
 python-ipmi==0.5.4
 python-openstackclient==6.5.0
 requests==2.32.2
 service_identity==24.1.0
-PyYAML==6.0
+PyYAML==6.0.1
 setuptools==65.5.1
 werkzeug==3.0.3
 wheel==0.42.0
 zope.interface==5.4.0
 
-git+https://github.com/krkn-chaos/arcaflow-plugin-kill-pod.git
+git+https://github.com/krkn-chaos/arcaflow-plugin-kill-pod.git@v0.1.0
```

Comment:
I see there is a pip package out there for this, is it not set up to properly update versions? 
https://pypi.org/project/arcaflow-plugin-kill-pod/

Reply:
Not sure if reflects the current build. I'm afraid that comes from when the plugin was in charge of the arca team. AFAIK the action is broken due to some arca related steps that are broken that's why we're fetching it from GitHub. I could fix the action eventually, but for the moment I'd stick to this workaround, thoughts?

Reply:
ok! that sounds good to me for now! dont want to block based on this